src/Controllers/VoyagerBreadController.php

```diff
@@ -246,6 +246,17 @@ public function getContentBasedOnType($request, $slug, $row){
       /********** FILE TYPE **********/
     } else if($row->type == 'file'){
 
+        $file = $request->file($row->field);
+        $filename = str_random(20);
+
+        $path =  $slug . '/' . date('F') . date('Y') . '/';
+
+        $full_path = $path . $filename . '.' . $file->getClientOriginalExtension();
+
+        Storage::put(config('voyager.storage.subfolder') . $path . $filename . '.' . $file->getClientOriginalExtension(), (string)$file, 'public');
```

Comment:
I just place the upload code from "image uploading". I just follow the current code and variable styles.

When I select InputType=file, there is a error and I just follow and tried with this code and file uploaded, as per media library folder and file pattern.